admin_guide/install/first_steps.adoc

```diff
@@ -64,21 +64,21 @@ link:../../using_images/db_images/mysql.html[*MySQL*], and
 link:../../using_images/db_images/postgresql.html[*PostgreSQL*] to support data
 storage.
 
-To create the core set of image streams, which use CentOS 7 based images:
+If your node hosts are subscribed using Red Hat Subscription Manager and you
+want to use the Red Hat Enterprise Linux (RHEL) 7 based images:
 
 ----
 $ oc create -f \
-    examples/image-streams/image-streams-centos7.json \
+    examples/image-streams/image-streams-rhel7.json \
     -n openshift
 ----
 
-Alternatively, if you are using node hosts that are subscribed using Red Hat
-Subscription Management and prefer to use the Red Hat Enterprise Linux (RHEL) 7
+Alternatively, to create the core set of image streams when using the CentOS 7
```

Comment:
The "when using" here is a little strange/misleading. The original ", which use CentOS 7 based images" was more accurate.